backend/scripts/nlp/nltk-json.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20c7909f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install textblob\n",
+    "!pip install -U pip setuptools wheel\n",
+    "!pip install -U spacy\n",
+    "!python -m spacy download en_core_web_sm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cacdd020",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "from os import path\n",
+    "from os.path import exists\n",
+    "import string\n",
+    "from collections import Counter\n",
+    "from tokenize import maybe\n",
+    "import nltk\n",
+    "import matplotlib.pyplot as plt\n",
+    "from textblob import TextBlob\n",
+    "from pathlib import Path\n",
+    "from textblob.sentiments import NaiveBayesAnalyzer\n",
+    "import nltk.corpus\n",
+    "from nltk.tokenize import word_tokenize\n",
+    "from nltk.probability import FreqDist\n",
+    "from nltk.corpus import stopwords\n",
+    "from nltk.sentiment.vader import SentimentIntensityAnalyzer\n",
+    "import nltk.corpus\n",
+    "from nltk.tokenize import RegexpTokenizer\n",
+    "from nltk.tokenize import word_tokenize\n",
+    "from nltk.data import load\n",
+    "fdist = FreqDist()\n",
+    "\n",
+    "# downloading required packages\n",
+    "nltk.download('nps_chat')\n",
+    "nltk.download('punkt')\n",
+    "nltk.download('brown')\n",
+    "nltk.download('movie_reviews')\n",
+    "nltk.download('stopwords')\n",
+    "nltk.download('vader_lexicon')\n",
+    "nltk.download('tagsets')\n",
+    "nltk.download('averaged_perceptron_tagger')\n",
+    "# nltk.help.upenn_tagset('MD')\n",
+    "\n",
+    "print('\\n'+'******************************************************************')\n",
+    "print('This application is about Natural Language Processing (NLP).')\n",
+    "print('The user enters sentence(s) add using NLP and other algorithm the application')\n",
+    "print('will analyze that entry and gives some feesback.')\n",
+    "print('\\n'+'*****   THIS APPLICATION DOES NOT SUPPORT CONTRACTED FORM    *****'+'\\n')\n",
+    "print('******************************************************************')\n",
+    "\n",
+    "\n",
+    "def main():\n",
+    "\n",
+    "    #     myPath = input('\\n'+'Enter the path for the file:')\n",
```

Comment:
There are multiple commented lines of code that do not appear to be comments? Should this be included or removed?